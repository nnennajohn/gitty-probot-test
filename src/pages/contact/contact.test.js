import React from 'react';

import ContactPage from './index';

const testIf =
  process.env.MODE !== undefined && process.env.MODE === 'skipsnapshots'
    ? test.skip
    : test;

testIf('Contact Page', () => {
  const wrapper = shallow(<ContactPage />);
  expect(wrapper).toMatchSnapshot();
});
