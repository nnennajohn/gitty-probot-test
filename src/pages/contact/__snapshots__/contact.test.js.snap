// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Page 1`] = `
<styled.div>
  <div
    className="contenidos_contacto_texto"
  >
    <p>
      <span
        className="contenidos_contacto_texto_titulo"
      >
        Santa
      </span>
      <br />
      <br />
      Torpedero Tucumán, 29
      <br />
      <br />
      Local Dcha.
      <br />
      <br />
      28016 - Madrid
      <br />
      <br />
      Tf: 
      <a
        href="tel:+34 91 353 39 54"
      >
        +34 91 353 39 54
      </a>
      <br />
      <br />
      <a
        href="mailto:info@santacc.es"
      >
        info@santacc.es
      </a>
    </p>
    <p
      className="contenidos_contacto_texto_metro"
    >
      Pío XII - Colombia
    </p>
  </div>
  <div
    className="contenidos_contacto_mapa"
  >
    <withScriptjs(withGoogleMap(CMap))
      center={
        Object {
          "lat": 40.463454,
          "lng": -3.67186,
        }
      }
      containerElement={
        <div
          style={
            Object {
              "height": "400px",
            }
          }
        />
      }
      googleMapURL="https://maps.googleapis.com/maps/api/js?key=AIzaSyDQ6tkAoMSO8ygMYmhuA9FrOTPVFJYKhqw"
      loadingElement={
        <div
          style={
            Object {
              "height": "100%",
            }
          }
        />
      }
      mapElement={
        <div
          style={
            Object {
              "height": "100%",
            }
          }
        />
      }
    />
  </div>
</styled.div>
`;
